build-tests/heft-action-plugin-test/README.md

```diff
@@ -1,6 +0,0 @@
-# heft-action-plugin-test
```

Comment:
Is this tested somewhere else?

Reply:
All plugins are technically "action plugins" now, since you can create a phase containing only one plugin. That is core functionality that is tested anywhere we use Heft, so would be covered by the Heft plugin test projects.